vast_pipeline/management/commands/clearpiperun.py

```diff
@@ -97,25 +95,30 @@ def handle(self, *args, **options) -> None:
 
         piperuns = options['piperuns']
         flag_all_runs = True if 'clearall' in piperuns else False
+
         if flag_all_runs:
-            logger.info('clearing all pipeline run in the database')
+            logger.info('Clearing all pipeline run in the database')
             piperuns = list(Run.objects.values_list('name', flat=True))
 
         for piperun in piperuns:
             p_run_name = get_p_run_name(piperun)
             try:
                 p_run = Run.objects.get(name=p_run_name)
             except Run.DoesNotExist:
-                raise CommandError(f'Pipeline run {p_run_name} does not exist')
+                raise CommandError("Pipeline run '%s' does not exist", p_run_name)
```

Comment:
You can leave this f-string - Its only the ones in `logging` calls that have issues :) 
```suggestion
                raise CommandError(f'Pipeline run {p_run_name} does not exist')
```